Backoffice/BusinessRules/Strategies/ContentTypeStrategy.php

```diff
@@ -2,22 +2,36 @@
 
 namespace OpenOrchestra\Backoffice\BusinessRules\Strategies;
 
+use OpenOrchestra\BaseBundle\Context\CurrentSiteIdInterface;
 use OpenOrchestra\ModelInterface\Model\ContentTypeInterface;
 use OpenOrchestra\Backoffice\Security\ContributionActionInterface;
 use OpenOrchestra\ModelInterface\Repository\ContentRepositoryInterface;
+use OpenOrchestra\ModelInterface\Repository\SiteRepositoryInterface;
 
 /**
  * class ContentTypeStrategy
  */
 class ContentTypeStrategy extends AbstractBusinessRulesStrategy
 {
+    CONST READ_LIST = 'READ_LIST';
```

Comment:
@alavieille : I'n not a big fan of having this const here, it should be better to have it in ContributionActionInterface or something similar to get it available for other kind of subjects